project/Versions.scala

```diff
@@ -18,5 +18,5 @@ object Versions {
 object PagopaVersions {
   lazy val commonsVersion                = "0.1.7"
   lazy val partyManagementVersion        = "0.1.x-SNAPSHOT"
-  lazy val partyProxyVersion             = "0.1.12"
+  lazy val partyProxyVersion             = "0.1.x-SNAPSHOT"
```

Comment:
Maybe I'm missing something, but updating this dependency could throw an exception at runtime.
Did you try to run this PR?

Reply:
Yes, in my local is working